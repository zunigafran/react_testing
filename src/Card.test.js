import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

test("Card component renders without errors", () => {
  const { container } = render(
    <Card caption="Test Caption" src="test-image.jpg" currNum={1} totalNum={3} />
  );

  expect(container).toMatchSnapshot();
});