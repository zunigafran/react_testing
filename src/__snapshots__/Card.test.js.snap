// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders without errors 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Test Caption
    </h4>
    <img
      alt="Test Caption"
      class="Card-image"
      src="test-image.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
